source/Jobbr.Server.nuspec

```diff
@@ -17,6 +17,13 @@
 		<dependency id="Jobbr.ComponentModel.JobStorage" version="[1.5,1.7)" />
 		<dependency id="Jobbr.ComponentModel.Management" version="[1.6,1.7)" />
 		<dependency id="Jobbr.ComponentModel.Registration" version="[1,1.1.2)" />
+		<dependency id="AutoMapper" version="[12.0.0,12.0.1)" />
+		<dependency id="Microsoft.Extensions.Configuration.Abstractions" version="[7.0.0,7.0.1)" />
+		<dependency id="Microsoft.Extensions.DependencyInjection" version="[7.0.0,7.0.1)" />
+		<dependency id="Microsoft.Extensions.Logging.Abstractions" version="[6.0.3,6.0.4)" />
+		<dependency id="NCrontab" version="[3.3.1,3.3.2)" />
+		<dependency id="SimpleInjector" version="[5.4.1,5.4.2)" />
+		<dependency id="TinyMessenger" version="[1.0.0,1.0.1)" />
```

Comment:
Can we expand the range?
Feel like it should be compatible with either all versions with the same major or at least minor version, so you could update to any wanted patch version.